doc/how-to-use-newconnector.md

```diff
@@ -15,7 +15,7 @@ Now you can create a Connector object by calling `NewConnector`, which creates a
 
 ```
 dsn := "sqlserver://username:password@hostname/instance?database=databasename"
-connector, err := mssql.NewConnector(dsn)
+connector, err := mssqldb.NewConnector(dsn)
```

Comment:
The package name is "mssql", the last element of the import path is mssqldb. The old code was correct.